wiki/Announcements/en.md

```diff
@@ -0,0 +1,10 @@
+# Announcements
```

Comment:
ugh, I will add them.. though it seems to be a new thing to do with these index pages :p